Obj2Tiles/Options.cs

```diff
@@ -46,6 +46,9 @@ public sealed class Options
     
     [Option("keep-intermediate", Required = false, HelpText = "Keeps the intermediate files (do not cleanup)", Default = false)]
     public bool KeepIntermediateFiles { get; set; }
+
+    [Option('t', "y-up-to-z-up", Required = true, HelpText = "Convert the upward Y-axis to the upward Z-axis, which is used in some situations where the upward axis may be the Y-axis or the Z-axis after the obj is exported.", Default = true)]
```

Comment:
It's a bit annoying that this is set to `Required = true` even though it has a default value. Is there a specific reason why it's required?

Reply:
Nice catch! I think it's not required